comms/discovery/db/queries/get_chat_messages.go

```diff
@@ -27,18 +27,24 @@ func ChatMessage(q db.Queryable, ctx context.Context, arg ChatMessageParams) (db
 
 // Get chat messages and reactions
 const chatMessagesAndReactions = `
-SELECT chat_message.message_id, chat_message.chat_id, chat_message.user_id, chat_message.created_at, chat_message.ciphertext, COALESCE(jsonb_agg(reactions) FILTER (WHERE reactions.message_id IS NOT NULL), '[]') AS reactions
+SELECT
+	chat_message.message_id,
+	chat_message.chat_id,
+	chat_message.user_id,
+	chat_message.created_at,
+	COALESCE(chat_message.ciphertext, '~' || chat_blast.plaintext) as ciphertext, -- todo: separate out ciphertext + plaintext... make ciphertext nullable
+
+	to_json(array(select row_to_json(r) from chat_message_reactions r where chat_message.message_id = r.message_id)) AS reactions
```

Comment:
interesting... why the switch?

also for consistency maybe capitalize the SQL keywords?